### DISCUSSION TITLE
Extended tweets for streaming

### DISCUSSION THREAD

**Discussion by Mazawrath** [tweepy/tweepy#1194] [Questions] [answered]
How would I set up a stream receive extended tweet instead of cutting of at `...`? I see the tweet_mode option but I can't figure out how to get that to work. I tried:
`myStream = tweepy.Stream(auth=api.auth, listener=myStreamListener, tweet_mode='extended')`
but this doesn't appear to work.

**Comment by Harmon758** [comment, marked as answer]
For the standard streaming API, Twitter provides extended Tweets with an `extended_tweet` field.
For example, you would access it using Tweepy with something like `status.extended_tweet["full_text"]`.
Note, however, this field won't exist for classic Tweets that are not extended.
For more information, see https://developer.twitter.com/en/docs/tweets/tweet-updates.
